.github/workflows/get-secretmanager-secrets-it.yml

```diff
@@ -0,0 +1,47 @@
+name: get-secretmanager-secrets Integration
+on: [push]
+jobs:
+  gcloud:
+    name: with setup-gcloud
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ./setup-gcloud
+      with:
+        service_account_email: ${{ secrets.SETUP_GCLOUD_IT_EMAIL }}
+        service_account_key: ${{ secrets.SETUP_GCLOUD_IT_KEY_B64 }}
+    - id: secrets
+      uses: ./get-secretmanager-secrets
+      with:
+        secrets: |-
+          token:${{ secrets.GET_SECRETMANAGER_SECRET_REF }}
+    - name: get output
+      run: echo '${{ steps.secrets.outputs.token }}'
```

Comment:
Do we want to be printing a secret to the build log?

Reply:
It's masked 😄but the previous action. Also, it's not a _real_ secret, so I'm not concerned about leaking it.